docs/examples/201/log-analytics-with-solutions-and-diagnostics/main.bicep

```diff
@@ -34,8 +34,9 @@ resource logAnalyticsWorkspace 'Microsoft.OperationalInsights/workspaces@2020-03
   })
 }
 
-resource logAnalyticsWorkspaceDiagnostics 'Microsoft.OperationalInsights/workspaces/providers/diagnosticSettings@2017-05-01-preview' = {
```

Comment:
>providers [](start = 84, length = 9)

Did we have a suppression for the workaround type?

Reply:
Yes - it's being removed in this PR.